runtime/vm/j9vm.tdf

```diff
@@ -561,6 +561,9 @@ TraceExit=Trc_VM_jitFillOSRBuffer_Exit Overhead=1 Level=3 Template="jitFillOSRBu
 TraceEntry=Trc_VM_sendResolveMethodHandle_Entry Overhead=1 Level=2 Template="sendResolveMethodHandle"
 TraceExit=Trc_VM_sendResolveMethodHandle_Exit Overhead=1 Level=2 Template="sendResolveMethodHandle"
 
+TraceEntry=Trc_VM_sendResolveConstantDynamic_Entry Overhead=1 Level=2 Template="sendResolveConstantDynamic"
```

Comment:
Tracepoints can only be added to the end of the file

Reply:
fixed